lib/contributionreward.js

```diff
@@ -149,7 +168,7 @@ class ContributionReward extends ExtendTruffleContract(SolidityContributionRewar
         reputationReward,
         ethReward,
         options.externalToken,
```

Comment:
option.externalToken should be removed .
It is passed as a separate param.

Reply:
This PR is about formatting, that is all. I would prefer to take note of other comments and address them elsewhere.